.github/workflows/ci-testing-deploy.yml

```diff
@@ -1086,6 +1086,8 @@ jobs:
           driver: docker
       - name: setup docker-compose
         run: sudo ./ci/github/helpers/setup_docker_compose.bash ${{ matrix.docker_compose }} ${{ matrix.docker_compose_sha }}
+      - name: setup rclone docker volume plugin
```

Comment:
not sure I understand why you need RClone in storage here? I do not see where it is used.

Reply:
wrong place. it's supposed to bin the `simcore.sdk`